src/Microsoft.AspNetCore.SignalR/SignalRAppBuilderExtensions.cs

```diff
@@ -3,13 +3,20 @@
 
 using System;
 using Microsoft.AspNetCore.SignalR;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.AspNetCore.Builder
 {
     public static class SignalRAppBuilderExtensions
     {
         public static IApplicationBuilder UseSignalR(this IApplicationBuilder app, Action<HubRouteBuilder> configure)
         {
+            var marker = app.ApplicationServices.GetService(typeof(SignalRMarkerService));
+            if (marker == null)
+            {
+                throw new InvalidOperationException("The SignalR service needs to be registered");
```

Comment:
Full stop at end of exception message.

Reply:
"Full stop" 😄

Reply:
Here in New Zealand we speak English as the Queen intended https://en.wikipedia.org/wiki/Full_stop

![](http://www.abc.net.au/news/image/8154612-3x2-700x467.jpg)